.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: ruby
 rvm:
-  - 2.3
```

Comment:
@pathawks looks like 2.3.1 is supported - https://travis-ci.org/jekyll/jekyll-import/jobs/192211050